stable/telegraf/Chart.yaml

```diff
@@ -12,5 +12,5 @@ keywords:
 home: https://www.influxdata.com/time-series-platform/telegraf/
 maintainers:
 - name: naseemkullah
-  email: naseemkullah@gmail.com
+  email: naseem@transit.app
```

Comment:
Please see https://github.com/helm/charts/blob/master/PROCESSES.md for deprecation guideline